src/node-lib/curriculum-api-2023/queries/pupilUnitListing/pupilUnitListing.schema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+import { syntheticUnitvariantsWithLessonIdsSchema } from "@oaknational/oak-curriculum-schema";
+
+import { ConvertKeysToCamelCase } from "@/utils/snakeCaseConverter";
+
+export const unitBrowseDataSchema = z.array(
+  syntheticUnitvariantsWithLessonIdsSchema.omit({
+    null_unitvariant: true,
```

Comment:
We might actually need the null_unitvariant when it comes to implementing optionality. I can't remember so this is fine for now.